### PR TITLE
Fix lowercase ctid 

### PR DESCRIPTION
**Pull request by cindyyan317**
I am thinking to make lowercase convert to string util function. Now the stringutils are for unittests only. Do you think we need to move  it out from unittests?

### CODE REVIEW

**Comment by godexsoft** [comment]
I'd rather add `toUpper` in JsonUtils which already features `toLower`. Arguably, JsonUtils is no longer the best name for the file though :) I don't think the StringUtils in unittests are very useful outside of unittests. Wdyt?

**Comment by cindyyan317** [comment]
> I'd rather add `toUpper` in JsonUtils which already features `toLower`. Arguably, JsonUtils is no longer the best name for the file though :) I don't think the StringUtils in unittests are very useful outside of unittests. Wdyt?

LOL I will squeeze it to JsonUtils then. Maybe we can change the filename to a more generic name 'Utilities.cpp' later?

File: src/web/impl/AdminVerificationStrategy.cpp [hunk 1]
```diff
@@ -39,7 +40,7 @@ PasswordAdminVerificationStrategy::PasswordAdminVerificationStrategy(std::string
     std::memcpy(sha256.data(), d.data(), d.size());
     passwordSha256_ = ripple::to_string(sha256);
     // make sure it's uppercase
-    std::transform(passwordSha256_.begin(), passwordSha256_.end(), passwordSha256_.begin(), ::toupper);
+    passwordSha256_ = util::toUpper(passwordSha256_);
```
Comment (kuznetsss) [inline src/web/impl/AdminVerificationStrategy.cpp]: You could move `passwordSha256_` here

**Review by kuznetsss — APPROVED** [review]
👍

**Pull request merged**